flink-runtime/src/test/java/org/apache/flink/runtime/clusterframework/TaskExecutorResourceUtilsTest.java

```diff
@@ -198,6 +198,35 @@ public void testConfigShuffleMemoryRange() {
 		});
 	}
 
+	@Test
+	public void testSanityCheckShuffleMemoryFractionLessThanMin() {
+		testSanityCheckShuffleMemoryFractionOutOfMinMaxRange(true);
```

Comment:
I'm not a huge fan of boolean parameters. I think it would be better to define an `enum` with values with a more descriptive name.